guides/doc-Provisioning_Guide/topics/proc_installing_ipv6_environment.adoc

```diff
@@ -0,0 +1,264 @@
+= Installing IPv6 environment
+
+{Project} allows provisioning of {RHEL} 8 or newer over IPv6 provisioning networks. Although provisioning templates which ship with {Project} include IPv6 support for PXE and HTTP (iPXE) scenarios, the only tested and certified workflow is UEFI HTTP Boot with unmanaged DHCPv6 server which is described in this chapter.
+
+== Limitations
+
+DHCP server in {RHEL} (ISC DHCP) does not provide an integration API for managing IPv6 records, therefore {SmartProxy} DHCP plugin that would provide DHCP management of IPv6 subnets does not exist. In this scenario, DHCP IPv6 must be deployed as a separate (unmanaged) service to boostrap client nodes into grub2 which then configures IPv6 networking either using DHCPv6 or via static IPv6 address.
+
+== Requirements
+
+ifeval::["{build}" == "satellite"]
+{SmartProxy} must be installed on {RHEL} version 7.9 or higher.
++
+{Project} or {SmartProxy} can be only installed on IPv6-only systems, dualstack is not supported.
```

Comment:
This sounds like a crucial thing, let me follow up with questions:
If users want to manage IPv6 networks, they must keep in mind that they can only do this if they manage ONLY IPv6 networks on Both {Project} and {SmartProxy}?
Is that possible to run {Project{ on an IPv4 network and {SmartProxy} on an IPv6 network? Or the whole setup including all proxies must be IPv6 only?

Reply:
Yes, we know this is bold limitation. We want to start with this for 6.8 and see if we can relax this in the future release. We can perhaps change the wording since the message we want to say is that we only test IPv6 setup on singlestack IPv6 installations where both Server and Proxy are IPv6 only (no dualstack). We can even relax this today with it is known to work, however we haven't verified all the features. Let's see if you can find something out of this :-)

Reply:
I am adding "known to work" note. This is an approved term by Satellite PMs.